kitsune/search/tests/test_search_simple.py

```diff
@@ -17,7 +17,7 @@
 
 
 def render(s, context):
-    t = jingo.env.from_string(s)
+    t = jingo.get_env().from_string(s)
```

Comment:
Looks like you upgraded to jingo 0.8.1 here. I haven't tested that with Puente and have no idea if that works. Let me know if you run into issues.